ports/openxlsx/portfile.cmake

```
@@ -0,0 +1,31 @@
+# Header-only library
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO troldal/OpenXLSX
+    REF "v${VERSION}"
+    SHA512 52205b394383d45c0fb16599ab96453d8a5b9b5cd596096848cc888f47565b2713d9edded06b2ecd7b67736622badf136e4b1becc57bfa5bbdcb1e063a347084
+    HEAD_REF master
+    PATCHES
+    compilation_fix.patch
+)
+set(CMAKE_CXX_STANDARD 17)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS
+        -DOPENXLSX_BUILD_TESTS=OFF
+)

```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH ${SOURCE_PATH}
    PREFER_NINJA
    OPTIONS
        -DCMAKE_CXX_STANDARD=17
        -DOPENXLSX_BUILD_TESTS=OFF
)
```
Portfiles run in script mode. `CMAKE_CXX_STANDARD` has no effect.
https://learn.microsoft.com/en-us/vcpkg/contributing/maintainer-guide#portfiles-are-run-in-script-mode